beesdoo_product/views/beesdoo_product.xml

```diff
@@ -72,6 +72,30 @@
         </field>
     </record>
 
+
+    <record model="ir.ui.view" id="beesdoo_product_tree">
+        <field name="name">bees.product.template.tree</field>
```

Comment:
Why don't you inherit from product.product_product_tree_view ?

Reply:
Because there are fields I want to remove, added in many different modules. And I want to add more. I figured it was easier and cleaner to start from scratch since this view is not meant to be inherited.

Should I rather move it to a `beesdoo_custom`  module? I heard of people doing that a the Odoo Experience.

Reply:
The action: product_normal_action_sell is explicit about the usage of product.product_product_tree_view. This mean if someone open this action, he will not get your specific list view. Since it's for product variant and the bees coop is not using them, I think i doesn't matter much.

Reply:
@houssine78 ok, I'll move it to a new module

@tfrancoi Ok thanks for the feedback